src/main/java/core/basesyntax/MainApp.java

```diff
@@ -8,9 +8,19 @@
  * машина начала свою работу.
  * - Переопределить метод stopWork() и вывести в консоль строку с сообщением что та или иная
  * машина прекратила свою работу.
- * - В классе Main создать добавить по одному екземпляру каждого класса.
- * - Отправить все машини на работу и остановить работу всех машин.
+ * - В классе Main создать по одному экземпляру каждого класса.
+ * - Отправить все машины на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
please see the message in the Slack channel and see the checklist.

Reply:
ok, thx you. I will change it